README.md

```diff
@@ -120,8 +115,7 @@ auger checksum -f <member-3-boltdb-file> -r 7
 # Oh noes! The checksum should have been the same!
 ```
 
-TODO
-----
+## TODO
```

Comment:
I think we can move these TODOs to the GitHub issues to get them discussed.

Reply:
Agreed - I've removed them from the file and opened issues.